samples/drivers/led/led_strip/src/main.c

```diff
@@ -32,9 +32,9 @@ LOG_MODULE_REGISTER(main);
 #define RGB(_r, _g, _b) { .r = (_r), .g = (_g), .b = (_b) }
 
 static const struct led_rgb colors[] = {
-	RGB(0x0f, 0x00, 0x00), /* red */
-	RGB(0x00, 0x0f, 0x00), /* green */
-	RGB(0x00, 0x00, 0x0f), /* blue */
+	RGB(0xff, 0x00, 0x00), /* red */
+	RGB(0x00, 0xff, 0x00), /* green */
+	RGB(0x00, 0x00, 0xff), /* blue */
```

Comment:
Looking at commit https://github.com/zephyrproject-rtos/zephyr/commit/1d2a551c65594b3e200adcc37b2f11294587e664, it doesn't seem to be an oversight:
` - make the pattern easier to look at by emitting less light`

Reply:
mmmm ok, kinda makes sense in a way, but then 15 (0x0f) on a max scale of 255 is actually _really_ low?

Reply:
I set that value by hand to make the adafruit strips not too bright when the sample was running, yeah.

Reply:
I updated the PR to make this configurable